code/modules/research/xenoarchaeology/artifact/artifact_necromarker.dm

```diff
@@ -0,0 +1,86 @@
+#define NECROMARKER_MIN_WHISPER_INTERVAL 200
+#define NECROMARKER_MAX_WHISPER_INTERVAL 450
+/obj/machinery/necromarker
+    name = "mysterious sculpture"
+    desc = "A mysterious scultpure of spiral spines, covered in runes."
+    icon = 'icons/xenoarch_icons/necromarker.dmi'
+    icon_state = "black"
+    density = 1
+
+    var/ticks_not_whispered = 0
+    var/next_whisper = 300
+    var/whispers = list("...bring me flesh...", "...make us whole...", "...we must be whole...", "...join us in unity...", "...one mind, one soul, one flesh...", "...MAKE US WHOLE...")
+    var/list/mob/dead/observer/candidates = list()
+
+    machine_flags = WRENCHMOVE
+
+/obj/machinery/necromarker/New()
+    . = ..()
+
+/obj/machinery/necromarker/MouseDrop_T(mob/M as mob, mob/user as mob)
+    if(!istype(M) || isobserver(user))
+        return
+    if(Adjacent(user))
+        Consume(M)
```

Comment:
You probably should add this behavior to attackby as well, so grabbing someone and hitting the thing with them consumes as well.